mario/tools/parsersclass.py

```diff
@@ -146,9 +167,25 @@ def parse_from_excel(
     mario.Database
     """
 
+    # check the inputs to be correct
```

Comment:
I think this error handling is too much. Knowing that the types are already listed in the function, we dont need to check every single of them. Some of them though I like it, like the mode.

Reply:
let's proceed with this way for now, we can improve later